airflow/providers/google/cloud/operators/vertex_ai/endpoint_service.py

```diff
@@ -81,6 +81,7 @@ def __init__(
         region: str,
         project_id: str,
         endpoint: Union[Endpoint, Dict],
+        endpoint_id: Optional[str] = None,
```

Comment:
Do you think this new parameter should be added to `template_fields` for the operator? I'm not sure if this would be useful or what kinds of values are typical for `endpoint_id`. Just sharing the idea.

Reply:
Also, this new parameter should be added to the operator's docstring so it can be included in the Python API docs.

Reply:
Thanks for your review! Added the new param to the docstring: https://github.com/apache/airflow/pull/23070/commits/60bc79439a060e5148670b74046a075ab61c23ad

Reply:
> Do you think this new parameter should be added to `template_fields` for the operator? I'm not sure if this would be useful or what kinds of values are typical for `endpoint_id`. Just sharing the idea.

Thanks for your suggestion! I don't think `endpoint_id` should be added to `template_fields`. I think the optional param `endpoint_id` is used only when users would like to create a Vertex Endpoint with a specified ID. I don't imagine an use case to specify the ID dynamically 😃